src/plugins/intel_cpu/src/nodes/experimental_detectron_generate_proposals_single_image.cpp

```diff
@@ -434,6 +331,23 @@ bool ExperimentalDetectronGenerateProposalsSingleImage::needPrepareParams() cons
     return false;
 }
 
+std::unique_ptr<jit_uni_nms_proposal_kernel> ExperimentalDetectronGenerateProposalsSingleImage::compileNMSKernel() {
+    jit_uni_nms_proposal_kernel::jit_nms_conf jcp { post_nms_topn_, nms_thresh_,
+        coordinates_offset };
+    std::unique_ptr<jit_uni_nms_proposal_kernel> nms_kernel;
+    if (mayiuse(avx512_core)) {
+        nms_kernel.reset(new jit_uni_nms_proposal_kernel_impl<avx512_core> { jcp });
+    } else if (mayiuse(x64::avx2)) {
+        nms_kernel.reset(new jit_uni_nms_proposal_kernel_impl<x64::avx2> { jcp });
+    } else if (mayiuse(sse41)) {
+        nms_kernel.reset(new jit_uni_nms_proposal_kernel_impl<sse41> { jcp });
+    } else {
+        IE_THROW() << "Unsupported ISA";
```

Comment:
Why throw exception instead of returning empty `std::unique_ptr<jit_uni_nms_proposal_kernel>` ?

Reply:
`compileNMSKernel()` method was removed in sake of `createPrimitive()` implementation.